src/jit/compiler.cpp

```diff
@@ -3525,7 +3525,7 @@ void Compiler::compInitOptions(JitFlags* jitFlags)
         }
         if (jitFlags->IsSet(JitFlags::JIT_FLAG_TIER1))
         {
-            printf("OPTIONS: Tier-1 compilation\n");
+            printf("OPTIONS: Tier-1 recompilation\n");
```

Comment:
It may be the first compilation with the tier 1 flag and may not be a recompilation (see other comment)
```suggestion
            printf("OPTIONS: Tier-1 compilation\n");
```